src/gatsby/models/blog/createPages.js

```diff
@@ -91,13 +75,19 @@ const createPages = async ({ graphql, actions }) => {
     throw blogResponse.errors
   }
 
+  const {
+    pageInfo: { itemCount },
+    tags,
+    nodes: posts
+  } = blogResponse.data.allBlogPost
+
   // Create home blog pages (with pagination)
   const blogHomeTemplate = path.resolve('./src/templates/blog-home.tsx')
 
   for (const page of pagesGenerator({
     basePath: '/blog',
     hasHeroItem: true,
-    itemCount: blogResponse.data.home.pageInfo.itemCount
+    itemCount: isDevMode ? 1 : itemCount
```

Comment:
it is useful! we should though make some file that engineers can change locally to set this, or at least read ENV var.

Reply:
why do we need to limit itemCount here btw?

Reply:
I was definitely thinking env var for a quick fix!
limiting itemCount is so only one blog index page is made. The blog indexes happen to make a lot of their own images with the thumbnails.

Reply:
I just don't quite understand (don't have time to dig) why do we limit number of posts in to two places (here and below) ... intuitively it should be one - at the very top of the processing